### PR TITLE
adds process id and kvfilter

### PR DESCRIPTION
**Pull request by przygienda**

### CODE REVIEW

**Comment by sile** [comment]
Could you tell me the state of this PR?
Is it ready for review?

**Comment by przygienda** [comment]
should be. I merged with newest stuff, tests pass

File: src/file.rs [hunk 1]
```diff
@@ -28,8 +33,13 @@ use std::time::{Duration, Instant};
 pub struct FileLoggerBuilder {
     common: BuilderCommon,
     format: Format,
+    source_location: SourceLocation,
```
Comment (sile) [inline src/file.rs:36]: The above `common: BuilderCommon` field contains `source_location`, `level`, `channel_size`, and `kvfilterparameters`. So it seems that we don't need to re-introduce those fields here.

**Review by sile — CHANGES_REQUESTED** [review]
As the first review, I left a minor comment.
Besides, it seems that [a CI lint check](https://github.com/sile/sloggers/pull/42/checks?check_run_id=2757568271) has failed, so could you please fix that?

**Comment by sile** [comment]
I'll take a more detailed look at this PR when I have time (I'll busy this week, so that may become next week or later).

**Comment by sile** [comment]
(I'm waiting for the CI failure to be resolved to restart PR review)

**Comment by sile** [comment]
Let me close this PR as there have been no activities for a long time.

**Pull request closed without merging**